src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -43,7 +44,7 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public Command parseCommand(String userInput) throws ParseException {
-        String trimmedUserInput = userInput.trim();
+        String trimmedUserInput = StringUtil.removeExtraWhitespace(userInput.trim());
```

Comment:
suggest maybe to use the trim function in the removeExtraWhitespace() function? So everything happens there